qa/smoke-test-http/src/test/java/org/elasticsearch/http/snapshots/RestGetSnapshotsIT.java

```diff
@@ -241,6 +242,62 @@ public void testFilterBySLMPolicy() throws Exception {
         );
     }
 
+    public void testSortAfterStartTime() throws IOException {
+        final String repoName = "test-repo";
+        AbstractSnapshotIntegTestCase.createRepository(logger, repoName, "fs");
+        final SnapshotInfo snapshot1 = AbstractSnapshotIntegTestCase.createFullSnapshot(logger, repoName, "snapshot-1");
+        final SnapshotInfo snapshot2 = AbstractSnapshotIntegTestCase.createFullSnapshot(logger, repoName, "snapshot-2");
+        final SnapshotInfo snapshot3 = AbstractSnapshotIntegTestCase.createFullSnapshot(logger, repoName, "snapshot-3");
+
+        final List<SnapshotInfo> allSnapshotInfo = clusterAdmin().prepareGetSnapshots(matchAllPattern())
+                .setSnapshots(matchAllPattern())
+                .setSort(GetSnapshotsRequest.SortBy.START_TIME)
+                .get()
+                .getSnapshots();
+        assertThat(allSnapshotInfo, is(List.of(snapshot1, snapshot2, snapshot3)));
+
+        final long startTime1 = snapshot1.startTime();
+        final long startTime2 = snapshot2.startTime();
+        final long startTime3 = snapshot3.startTime();
+
+        assertThat(allAfterStartTimeAscending(startTime1 - 1), is(allSnapshotInfo));
+        assertThat(allAfterStartTimeAscending(startTime1), is(allSnapshotInfo));
+        assertThat(allAfterStartTimeAscending(startTime2), is(List.of(snapshot2, snapshot3)));
```

Comment:
Are we sure that `snapshot1` and `snapshot2` get different timestamps?

Reply:
Maybe not on Windows where the clock is less accurate actually ... let me make sure they will.

Reply:
Adjusted the code to ensure the timestamps never collide now.

Reply:
Oh right missed this one, adding a fix here as well :)